.github/workflows/benchmark.yml

```diff
@@ -85,6 +85,7 @@ jobs:
             { echo "::error::Failed to find merge base"; exit 1; }
           echo "Merge Base: $merge_base"
           git checkout $merge_base
+          git submodule update --init --recursive
```

Comment:
this was previously not an issue because the checkout action did it but as the contender is missing them this step is needed to make it more robust.